src/styles/main.scss

```
@@ -233,3 +261,41 @@ input[type="datetime-local"]::-webkit-calendar-picker-indicator {
 .fetch-control.opened .fetch-control_toggle::after {
     content: '△';
 }
+
+.modal {
+    display: none;
+    position: fixed;
+    left: 50%;
+    top: 50%;
+    transform: translate(-50%, -50%);
+    max-width: 700px;
+    padding: 15px;
+    background: #fff;
+    border-radius: 6px;
+    box-shadow: 0 0 30px 30px rgb(0 20 36 / 10%);
+}
+
+.modal__header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.modal_close {
+    cursor: pointer;
+}
+
+.modal__title {
+    font-size: 18px;
+}
+
+.modal__subtitle {
+    font-size: 14px;
+}
+
+.modal__footer {
+    display: flex;
+    align-items: center;
+    justify-content: end;
```

```suggestion
    justify-content: flex-end;
```

Newer versions of scss (see #129) will complain with "(287:3) autoprefixer: end value has mixed support, consider using flex-end instead"